api/src/controllers/jobs.py

```diff
@@ -1,40 +1,39 @@
 import json
-from flask import Blueprint, Response
 
+from fastapi import APIRouter
 from config import config
-from enums import STATUS_CODES
 from use_case.delete_job import DeleteJobUseCase
 from use_case.get_result_job import GetResultJobUseCase
 from use_case.start_job import StartJobUseCase
 from use_case.status_job import StatusJobUseCase
+from starlette.responses import JSONResponse
+
+router = APIRouter(tags=["Jobs"], prefix="/job")
 
-blueprint = Blueprint("jobs", __name__)
 
 if config.JOB_SERVICE_ENABLED:
 
-    @blueprint.route("/api/job/<path:job_id>", methods=["POST"])
+    @router.post("/{job_id:path}", operation_id="start_job")
     def start(job_id: str):
         use_case = StartJobUseCase()
-        response = use_case.execute({"job_id": job_id})
-        return Response(json.dumps(response.value), mimetype="application/json", status=STATUS_CODES[response.type])
+        return use_case.execute({"job_id": job_id})
```

Comment:
We should align the use case logic with the template. We can do that in a separate PR.

Reply:
good idea, will create a new issue for this